apps/mark-scan/frontend/src/pages/diagnostics/diagnostics_screen.test.tsx

```diff
@@ -218,3 +226,58 @@ test('ending paper handler diagnostic refetches the diagnostic record', async ()
   await screen.findByText(/Test passed/);
   expectDiagnosticResult(screen, DiagnosticSectionTitle.PaperHandler, true);
 });
+
+test('PAT diagnostic success', async () => {
+  renderScreen();
+
+  userEvent.click(await screen.findButton('Test PAT Input (Sip & Puff)'));
+  apiMock.setPaperHandlerState('pat_device_connected');
+  await screen.findByText(
+    'Personal Assistive Technology Device Identification'
+  );
+
+  // Continue past intructions
+  userEvent.keyboard('1');
```

Comment:
nit: could we replace the raw strings with [`Keybindings.[MOVE|SELECT]`](https://github.com/votingworks/vxsuite/blob/main/libs/ui/src/keybindings.ts#L23-L24)? (probably also worth updating the [`behaviorToKeypressMap`](https://github.com/votingworks/vxsuite/blob/main/apps/mark-scan/frontend/src/pages/pat_device_identification/constants.ts#L1) used in the PAT identification flow - missed that when I was consolidating the keybindings recently)